packages/typescript/server/package.json

```diff
@@ -37,7 +37,6 @@
     "url": "https://github.com/jamsocket/jamsocket/issues"
   },
   "dependencies": {
-    "isomorphic-fetch": "^3.0.0",
-    "server-only": "^0.0.1"
```

Comment:
do we not want to keep this?

Reply:
Yeah, removing the `server-only` package is intentional. Turns out that package is specific to nextjs, so some folks were seeing server-only errors when running outside of nextjs